pkgs/development/python-modules/capstone/default.nix

```diff
@@ -1,21 +1,24 @@
-{ lib, stdenv
+{ stdenv
+, lib
 , buildPythonPackage
-, fetchPypi
+, capstone
 , fetchpatch
+, fetchPypi
 , setuptools
-, capstone
 }:
 
 buildPythonPackage rec {
   pname = "capstone";
-  version = stdenv.lib.getVersion capstone;
+  version = lib.getVersion capstone;
 
   src = capstone.src;
   sourceRoot = "${capstone.name}/bindings/python";
 
   postPatch = ''
     ln -s ${capstone}/lib/libcapstone${stdenv.targetPlatform.extensions.sharedLibrary} prebuilt/
     ln -s ${capstone}/lib/libcapstone.a prebuilt/
+  '' + lib.optionalString stdenv.targetPlatform.isAarch64 ''
+    substituteInPlace setup.py --replace manylinux1 manylinux2014
```

Comment:
I thought about this a bit more and do you know if it would be save to apply on all platforms? Otherwise it might be forgotten with the next update.

Reply:
@SuperSandro2000: Indeed, that makes sense. I removed the conditional entirely. PTAL